.github/workflows/add-model-like.yml

```diff
@@ -37,8 +37,10 @@ jobs:
           pip install -e .[dev]
 
       - name: Check transformers location
+        # make `transformers` available as package (required since we use `-e` flag) and check it indeeds from the repo.
         run: |
           . ~/venv/bin/activate
+          python setup.py build install
```

Comment:
@sgugger let me know if you have any comment regarding this new change.

(Lysandre and gante are OK with it on a Slack message)

Reply:
Sounds good!